server/models.py

```diff
@@ -20,7 +20,7 @@
 
 import contextlib
 import csv
-from datetime import datetime as dt
+import datetime as dt
```

Comment:
Context for change: This style is suggested by [the style guide](https://github.com/amontalenti/elements-of-python-style#the-standard-library)